src/test/java/io/logz/logback/LogzioLogbackAppenderTest.java

```diff
@@ -90,7 +90,7 @@ public void validateJsonMessage(){
     }
 
     @Test
-    public void simpleAppending() throws Exception {
```

Comment:
Why we need to remove `throws Exception` ?

Reply:
I removed it from all the methods that don't throw any exceptions (that had warning on them `Exception 'java.lang.Exception' is never thrown in the method`)